src/side-navigation/side-navigation.scss

```diff
@@ -170,3 +171,71 @@
     }
   }
 }
+
+@mixin iui-side-navigation-submenu {
+  display: none;
+  visibility: hidden;
+  opacity: 0;
+  width: 0;
+  height: 100%;
+  box-sizing: border-box;
+  padding: 0 $iui-sm;
+  overflow-x: hidden;
+  overflow-y: overlay;
```

Comment:
Add `overflow-y: auto` before this rule to support all browsers.

Reply:
Done.

Reply:
I meant like this:
```scss
overflow-y: auto;
overflow-y: overlay;
```

This will make sure that `overlay` is used when supported, and `auto` is used as fallback.

Reply:
Ah, ok, I misunderstood.  Implemented like you suggested now.